### ISSUE TITLE
Computed properties still can't fully express all dependencies. Consider some mechanism of imperatively directing them to update

### ISSUE THREAD

**Issue by sjmiles** [Polymer/polymer-expressions#31]
One can use filter syntax to conveniently construct computed properties. For example, I made a json-dump element that uses this construction:

`{{object | stringify}}`

Where `stringify` is a method on my element which stringifies an input object. 

However, the stringified representation of `object` depends on more than `object` (iow, my filter has non-explicit dependencies on `object`'s properties). 

It's normal behavior for the binding not to update when `object`'s properties changes, so it would be useful if there was a way to touch, tickle, or kick the binding explicitly when needed.

**Comment by dzhioev** [comment]
Hello @nomego, did you find a proper solution for your problem?
I ask, because we bumped into the same problem with our filter named 'i18n'.

**Comment by nomego** [comment]
@dzhioev No, we haven't looked into it much since, actually. Let me know if you figure something out!

**Comment by donwojtallo** [comment]
Hi there.
The only way to invalidate expression I've found is to add language as parameter. I know it's an obvious solution, but i could not find any other way.
You can set a global variable:
```var l='en';```
And then use this expression:
```{{ [message,l] | t }}```
Or this:
```{{ message | t(l) }}```
If you change 'l' or 'message' the expression will update.
It does not solve the problem, but it's good workaround for me. I hope there will be better solution in the future.
